include/plugin/zeek-compat.h

```diff
@@ -8,7 +8,11 @@
 
 #include <zeek/zeek-config.h>
 
-#ifdef ZEEK_VERSION_NUMBER
+//// Include zeek/zeek-version.h when building internally with latest Zeek.
+#ifndef ZEEK_VERSION_NUMBER
```

Comment:
Let's instead directly check for the header,
```suggestion
#if __has_include("<zeek/zeek-version.h>")
```